src/elements/Accordion/Accordion.js

```diff
@@ -90,8 +90,8 @@ class Accordion extends Component {
 				toggle: 'hide'
 			}, () => {
 				window.requestAnimationFrame(() => {
-					const transitionDuration = Math.max(this.accordionContent.current.clientHeight, 300) > 5000
-						? 5000
+					const transitionDuration = Math.max(this.accordionContent.current.clientHeight, 300) > 2000
+						? 1000
```

Comment:
this logic makes no sense
if it's gonna take longer than 2 seconds, it'll make it 1 second. else everything else can go up to 2.
both numbers should be `2000`

Reply:
Isn't it if the client height is more than 2000, make it 1 second? Otherwise, make it whatever seconds the height it is? (height of container, 300px = 300ms)